sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluators/_tool_relevance/_tool_relevance.py

```diff
@@ -0,0 +1,256 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+import os
+import logging
+import math
+from typing import Dict, List, Union, TypeVar, Optional
+from typing_extensions import overload, override
+from azure.ai.evaluation._evaluators._common import PromptyEvaluatorBase
+from azure.ai.evaluation._exceptions import (
+    ErrorBlame,
+    ErrorCategory,
+    ErrorTarget,
+    EvaluationException,
+)
+from ..._common.utils import check_score_is_valid, reformat_conversation_history
+from azure.ai.evaluation._common._experimental import experimental
+
+logger = logging.getLogger(__name__)
+
+T_EvalValue = TypeVar("T_EvalValue")
+
+
+@experimental
+class ToolRelevanceEvaluator(PromptyEvaluatorBase[Union[str, float]]):
```

Comment:
I thought we'll start to add evaluators to the new evaluator catalog. Why do you still want to add this evaluator here?